addons/forum-post-countdown/addon.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Cooldown display for posting in the forums",
+  "description": "After submitting a forum post, display a countdown showing when you can post again.",
+  "tags": ["community", "forums"],
+  "versionAdded": "1.33.0",
+  "userscripts": [
+    {
+      "url": "post-create.js",
+      "matches": ["newPostScreens"]
+    },
+    {
+      "url": "userscript.js",
+      "matches": ["https://scratch.mit.edu/*"]
+    }
```

Comment:
Why does this need to run on every page?

Reply:
So the cooldown is shown on every page.

Reply:
Is that really necessary?

Reply:
I was thinking just post buttons in the forums?

Reply:
That probably does make sense.